packages/holocron-module-route/package.json

```diff
@@ -29,9 +29,9 @@
     "holocron": "^1.0.0-4"
   },
   "dependencies": {
-    "@americanexpress/one-app-router": "1.0.0",
+    "@americanexpress/one-app-router": "^1.1.0",
```

Comment:
were there any more changes to one-app-router apart from that function to add the `matchPromise`? what prompted this lib update?

Reply:
the updated version also removed some of the unsafe lifecycle methods